### ISSUE TITLE
WARNING: MLS version 9.4.7.25 not supported

### ISSUE THREAD

**Issue by hfleitas** [MicrosoftDocs/sql-docs#2207]
Made a few edits to Install-MLModels2019ctp3.ps1

line 51:                     $this.RSetupPath = "$($this.RootPath)\Setup Bootstrap\SQL2019CTP3.0\x64\RSetup.exe" -f (ConvertSqlVersionToName $sqlVersion)

line 60:             $this.RSetupPath = "{0}\SQL2019CTP3.0\x64\RSetup.exe" -f`

lines 105-108:         elseif ($this.MlsVersion.ToString(3) -eq "9.2.1"`
            -or $this.MlsVersion.ToString(3) -eq "9.3.0"`
            -or $this.MlsVersion.ToString(3) -eq "9.4.1"`
			-or $this.MLsVersion.ToString(3) -eq "9.4.7")

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 125dc74a-69d9-807f-dfce-a3d7065be7c7
* Version Independent ID: e2401bd1-7bfe-3b8a-fd38-9f03f3f701bb
* Content: [Install pre-trained machine learning models - SQL Server Machine Learning - SQL Server](https://docs.microsoft.com/en-us/sql/advanced-analytics/install/sql-pretrained-models-install?view=sql-server-ver15#feedback)
* Content Source: [docs/advanced-analytics/install/sql-pretrained-models-install.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/advanced-analytics/install/sql-pretrained-models-install.md)
* Product: **sql**
* Technology: **machine-learning**
* GitHub Login: @dphansen
* Microsoft Alias: **davidph**

**Comment by dphansen** [comment]
@hfleitas Thank you for submitting this issue. I will investigate and get back to you.

**Comment by daleq** [comment]
@hfleitas Thanks.  I'll look into this and give you an response tomorrow.

**Comment by dphansen** [comment]
@hfleitas We are still investigating the issue. Will take a bit longer. Will keep you posted.

**Comment by MightyPen** [comment]
@dphansen Hi David, A minor info: We no longer use any of the labels whose names begin with "Issue-*".
So I have removed that label from this Issue 2207, and I have added the label "assigned-to-author".
Thanks. (GeneMi)

**Comment by dphansen** [comment]
@hfleitas Apologies for the delay. We are still investigating the issue.

**Comment by dphansen** [comment]
@hfleitas The product team is currently working on an update of the **Install-MLModels.ps1** script to support SQL Server 2019. When ready, it will be available on the docs article mentioned in this GitHub issue. Thank you for your feedback. It is really appreciated.

**Comment by dphansen** [comment]
@hfleitas A new version of the script can now be found at https://aka.ms/mlm4sql (also linked from the [docs article](https://docs.microsoft.com/en-us/sql/advanced-analytics/install/sql-pretrained-models-install?view=sql-server-ver15)).
